Frameworks/UIKit/UIImage.mm

```diff
@@ -889,22 +892,34 @@ + (UIImage*)animatedResizableImageNamed:(NSString*)name capInsets:(UIEdgeInsets)
     return StubReturn();
 }
 
+static NSData* _ObtainImageRepresentation(UIImage* img, CFStringRef type, NSDictionary* properties) {
+    RETURN_NULL_IF(!img);
+    CFMutableDataRef cfData = CFDataCreateMutable(nullptr, 0);
+    CFAutorelease(cfData);
+    CGImageDestinationRef destination = CGImageDestinationCreateWithData(cfData, type, 1, nullptr);
+    CFAutorelease(destination);
+    CGImageDestinationAddImage(destination, getImage(img), static_cast<CFDictionaryRef>(properties));
+    if (!CGImageDestinationFinalize(destination)) {
+        return nil;
+    }
+    return static_cast<NSData*>(cfData);
+}
+
 /**
- @Status Stub
+ @Status Interoperable
 */
 NSData* UIImagePNGRepresentation(UIImage* img) {
```

Comment:
i am not clear if it is a bug when you say I have test for it but looks like our PNG representation ends up being a bit off vs the actual PNG data on disk or simply we need regenerate the test artifact so that the test can pass? 
when you say update the CR, i was expecting another iteration potentially with above test update? 

---
In reply to: [96090810](https://github.com/Microsoft/WinObjC/pull/1704#discussion_r96090810) [](ancestors = 96090810)